guides/common/modules/proc_registering-hosts.adoc

```diff
@@ -16,14 +16,14 @@ endif::[]
 * You must have root privileges on the host that you want to register.
 ifdef::satellite,orcharhino[]
 * You must have an activation key created.
-* Optional: If you want to register hosts to Red{nbsp}Hat Insights, you must synchronize the `{RepoRHEL7Server}` repository and make it available in the activation key that you use.
-This is required to install the `insights-client` package on hosts.
 * {ProjectServer}, any {SmartProxyServer}s, and all hosts must be synchronized with the same NTP server, and have a time synchronization tool enabled and running.
 * The daemon *rhsmcertd* must be running on the hosts.
 * An activation key must be available for the host.
 For more information, see {ContentManagementDocURL}Managing_Activation_Keys_content-management[Managing Activation Keys] in _{ContentManagementDocTitle}_.
-* Subscription Manager must be version 1.10 or later.
-The package is available in the standard {RHEL} repository.
+endif::[]
+ifdef::satellite[]
+* Optional: If you want to register hosts to Red{nbsp}Hat Insights, you must synchronize the `{RepoRHEL7Server}` repository and make it available in the activation key that you use.
```

Comment:
Unrelated to your PR, but I wonder why it mentions the RHEL 7 repo. I'd think it's actually the RHEL version that the client is running.

Reply:
I believe RepoRHEL8BaseOS and RepoRHEL8AppStream together replace the RepoRHEL7Server on RHEL 8.

Reply:
Applied your suggestion in the last commit.